src/status_im/chat/models/message.cljs

```diff
@@ -287,35 +287,53 @@
   (apply fx/merge cofx
          (map (partial update-last-message (:chats db)) chat-ids)))
 
+(fx/defn unfresh-public-chats
```

Comment:
probably I would go for slightly different naming, rather than `unfresh`/`fresh`, I would probably change `fresh` to `never-synced` (as that is what they are correct?), fresh is a bit harder as `synced` might be a bit misleading, and `synced-at-least-once` is probably too verbose, but minor point

Reply:
yes, fresh means never-synced with mailserver or never received any messages from other client peers active in the chat after chat (re)added.

I went ahead and re-factored the naming. Also re-factored duplication and unnecessary filtering steps in chats->never-synced-public-chats.